### DISCUSSION TITLE
Is using CORS with Falco currently possible?

### DISCUSSION THREAD

**Discussion by CameronThornton** [falcoframework/Falco#97] [Q&A] [answered]
Hey there, I'm just curious if it's currently possible to use CORS in a Falco project?  If not, I'm curious if that's on the list?  Thank you.

**Comment by pimbrouwers** [comment, marked as answer]
Sure is! Anything possible with ASPNET Core is possible with Falco. Here is an example using the host builder:

```fsharp
webHost [||] {
        // rest...
        add_service (fun svc -> 
            svc.AddCors(fun o -> 
                o.AddPolicy(name = "PolicyName", configurePolicy = fun policy ->
                    policy.WithOrigins("https://example.com") |> ignore)))

        use_middleware (fun app ->
            app.UseCors("PolicyName"))

        // rest...
}
```

**Reply by CameronThornton** [reply]
Oh wow thanks so much.  Can't believe I missed the add_service feature.

Thanks again!

**Comment by pimbrouwers** [comment]
My pleasure! Thanks for reaching out!
